tools/autograd/templates/python_torch_functions.cpp

```diff
@@ -201,23 +200,12 @@ static PyObject * THPVariable_clamp(PyObject* module, PyObject* args, PyObject*
 
   ParsedArgs<4> parsed_args;
   auto r = parser.parse(args, kwargs, parsed_args);
-  if (!r.isNone(1) && !r.isNone(2)) {
+  if (!r.isNone(1) || !r.isNone(2)) {
```

Comment:
I still leave some check code here is that I want to avoid user passing Nones to both min and max, something like `clamp(a, None, None)`,  here we could throw a runtime error to the user for such use cases. The wrapper generator could not generate this condition check yet. Do you think it's appropriate we change the wrapper generator to address this condition?

Reply:
Or another solution could be: we treat user inputs like `clamp(a, None, None)` as a total valid inputs, we did nothing when we receive such calls in Aten native. This way we can remove the special case handling all together in frontend, and only do the check within Aten native. Should this be a better solution?